### DISCUSSION TITLE
Issue encountered while using "git"

### DISCUSSION THREAD

**Discussion by linghuccc** [community/community#132117] [Programming Help]
### Body

I was trying to upload my project to github, but encountered big size file error. The issue is that the file is actually ignored in the .gitignore file. As can be seen from below, the ".yarn" directory is grey.

![git1](https://github.com/user-attachments/assets/5b054320-4034-457f-999c-29bacfd6cedb)

Also, I have run "git ls-files", and the file is not even in the list.

![git2](https://github.com/user-attachments/assets/ad2994d0-3785-4c32-b0e9-a70a3193981d)

I wonder where the error comes from. Please advise. Thank you.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by GreeningSiren** [comment]
Hello,

Git will not stop tracking a file that is already being tracked, even if you add it to . gitignore

You can use the command
`git rm --cached -r .yarn` to tell git to stop tracking the directory. 
Then you can commit your changes

If this has helped you, mark it as the answer using the `Mark as answer` button

- GreeningSiren

**Reply by linghuccc** [reply]
Yes, I have done that. That is exactly why I have shown result of `git ls-files`. If I have not done so, it will be shown on the file list.

By the way I think you are referring to `git rm -–cached -r .yarn`, right?

**Reply by GreeningSiren** [reply]
Ooh, I see that's weird 🤔

And yes I was referring to that, but missed the extra  `-`
